src/main/java/bio/terra/service/snapshotbuilder/utils/CriteriaQueryBuilder.java

```diff
@@ -101,31 +125,25 @@ FilterVariable generateFilter(SnapshotBuilderDomainCriteria domainCriteria) {
     TableVariable ancestorVariable =
         TableVariable.forJoined(
             ancestorPointer,
-            "ancestor_concept_id",
+            DESCENDANT_CONCEPT_ID,
             new FieldVariable(
                 new FieldPointer(occurrencePointer, domainOption.getColumnName()),
                 occurrenceVariable));
 
     return new SubQueryFilterVariable(
-        getFieldVariableForRootTable(PERSON_ID_FIELD_NAME),
+        getFieldVariableForRootTable(PERSON_ID),
         SubQueryFilterVariable.Operator.IN,
         new Query(
             List.of(
                 new FieldVariable(
-                    new FieldPointer(occurrencePointer, PERSON_ID_FIELD_NAME), occurrenceVariable)),
+                    new FieldPointer(occurrencePointer, PERSON_ID), occurrenceVariable)),
             List.of(occurrenceVariable, ancestorVariable),
             new BooleanAndOrFilterVariable(
                 BooleanAndOrFilterVariable.LogicalOperator.OR,
                 List.of(
                     new BinaryFilterVariable(
                         new FieldVariable(
-                            new FieldPointer(occurrencePointer, domainOption.getColumnName()),
-                            occurrenceVariable),
-                        BinaryFilterVariable.BinaryOperator.EQUALS,
-                        new Literal(domainCriteria.getConceptId())),
```

Comment:
Just curious, can you explain why we need this change?

Reply:
We don't need this because every concept is its own parent

Reply:
Is this change correct? Before it was
```
(occurrence.concept_id = domain_concept_id OR
 concept_ancestor.ancestor_concept_id = domain_concept_id)
```
and now it's
```
(concept_ancestor.ancestor_concept_id = domain_concept_id)
```

Maybe I'm misreading this but it looks like the occurrence table is no longer referenced in the WHERE clause. If so, then the SELECT will select all rows in the occurrence table.

Can you provide the entire SQL for this function (`generateFilter(SnapshotBuilderDomainCriteria)`) before and after this change?

Reply:
I see, I missed that the `forJoined` above is joining on the occurrence table